pheps/phep-9999.md

```diff
@@ -0,0 +1,162 @@
+- Within six months of PyHC website update (written communication provided in email form), packages must submit a PR to [the PyHC website GitHub](https://github.com/heliophysicsPy/heliophysicsPy.github.io) indicating what level they believe their package is
+    - need to include written justification on the PR for chosen tier 
+- Within three months of PR publish, PyHC Tech Lead reviews the package and asserts if self-evaluated level matches expectations
+    - again, providing written reasoning on the PR if a tier regrade is deemed necessary
+- PyHC Lead reviews and gives final vote of approval on tier selection
+- PyHC Lead then either merges the PR, or provides written justification on the PR if a tier upgrade is deemed necessary  
+
+General comments:  
+
+If a package does not submit a self evaluation within six months of PyHC website update, they are automatically placed into the Copper tier. They will be able to apply for a tier regrade if this occurs, using the same process listed above.  
+
+Packages are allowed to move between tiers. If a package upgrades their status to match that of Silver, instead of Bronze, tier, for example, they can submit a new PR to have their tier regraded.   
+
+The flip side is also true; should a package become defunct or drop in status, they may be downgraded to a lower tier. Should this occur:  
+- Packages will receive six months' notification before a tier regrade is to take place through written e-mail communication (from the PyHC Lead)  
+- Packages then have the opportunity to rectify issues within the six-month period, through changes submitted in a PR  
+    - If changes result in continued compliance with current tier (as deemed by the PyHC Tech Lead and PyHC Lead), no tier regrade happens  
+    - If changes do not result in continued compliance with current tier, a tier regrade happens; packages may later on apply for a tier regrade once more  
+
+# Backwards Compatibility
+<a name="backwards-compatibility"></a>
+This PHEP does not propose a direct change to PyHC package code, simply the inclusion or not of packages within the various tiers, thus it introduces no compatibility concerns.
+
+# Security Implications
+<a name="security-implications"></a>
+This PHEP raises no security implications as it does not interact with any executing code.
+
+# How to Teach This
+<a name="how-to-teach-this"></a>
+This PHEP's contents and changes will be presented on, discussed, and hacked at various PyHC bi-weekly telecons and PyHC bi-annual meetings. Additionally, explanations for tiering, the process of obtaining a PyHC tier, etc. will be posted on the new main Projects page, as well as communicated within a blog post under the PyHC Blog page.   
+
+Further, links to pages (and potentially recordings) that explain how to satisfy each requirement for each tier will be made and communicated through a PyHC blog post (link then emailed to the community).  
```

Comment:
Does this need further definition with the PHEP, or is the promise of documentation sufficient?

Reply:
A note that this will supersede the existing [project submission process](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_pyhc_projects/adding_to_pyhc_project_list.md) is probably helpful. Also potentially a list of differences:

- Self evaluation is now just starting point, TSC evaluation is required
- Additional requirements beyond the main PyHC standards

Plus, of course, a commitment to update the submission process.

Reply:
Noted!